lib/widgets/picos_radio_select.dart

```diff
@@ -40,11 +40,14 @@ class PicosRadioSelect extends StatefulWidget {
 class _PicosRadioSelectState extends State<PicosRadioSelect> {
   dynamic _selectValue;
 
+  final double _distance = 15;
```

Comment:
```suggestion
  static const double _distance = 15;
```
One more optimization.